packages/web3-eth-contract/src/contract.ts

```diff
@@ -905,16 +914,25 @@ export class Contract<Abi extends ContractAbi>
 
 				abi.payable = abi.stateMutability === 'payable' ?? abi.payable;
 
+				const contractMethod = this._createContractMethod(abi, errorsAbi);
+
+				if (!this._overloadedMethodAbis.get(abi.name)) {
+					this._overloadedMethodAbis.set(abi.name, []);
+				}
```

Comment:
What is this if is all about? I see that in both cases `this._overloadedMethodAbis.set(abi.name, ...)` is called later.

Reply:
Yes, but I see that you handle this down with `...(this._overloadedMethodAbis.get(abi.name) ?? [])`, right?

Reply:
yes. thanks. removed